sklearn/metrics/ranking.py

```diff
@@ -22,6 +22,7 @@
 import warnings
 import numpy as np
 from scipy.sparse import csr_matrix
+import pandas
```

Comment:
Hi, thanks for the PR. But AFAIR, it is a convention in general to import pandas and also put it in a try catch block ( like [here](https://github.com/scikit-learn/scikit-learn/blob/master/sklearn/tests/test_grid_search.py#L491) ). I suppose this is necessary since Pandas is not a compulsory requirement for installing scikit-learn and this import would raise an error in case Pandas isn't installed. Other than that, LGTM.